src/targets/radiomaster_2400.ini

```diff
@@ -21,3 +21,24 @@ extends = env:RadioMaster_TX16S_2400_TX_via_ETX
 
 [env:RadioMaster_TX16S_2400_TX_via_WIFI]
 extends = env:RadioMaster_TX16S_2400_TX_via_ETX
+
+[env:RadioMaster_Ranger_2400_TX_via_UART]
+extends = env:Unified_ESP32_2400_TX_via_UART
+board_config = radiomaster.tx_2400.ranger
+
+[env:RadioMaster_Ranger_2400_TX_via_WIFI]
+extends = env:RadioMaster_Ranger_2400_TX_via_ETX
```

Comment:
`RadioMaster_Ranger_2400_TX_via_ETX` is incorrect. Should be `RadioMaster_Ranger_2400_TX_via_UART`.

Reply:
Good catch, I completely missed that one. I might have to extend the validator 😉